Assets/Scripts/BlackHolePull.cs

```diff
@@ -30,6 +30,9 @@ private void OnTriggerExit(Collider other)
 
     private void FixedUpdate()
     {
+        // first filter out any pulled bodies that may have been destroyed since the last update
```

Comment:
We'd need to performance test this. But thanks for adding this in.

Reply:
Is it good for now though? I didn't notice any performance issues.

Reply:
Yeah it's probably fine but maybe not great when there are a ton of asteroids being pulled all the time, and this is running every fixed update